frontend/src/pages/DashboardPage/Dashboard/Dashboard.jsx

```diff
@@ -7,6 +7,7 @@ import "./Dashboard.css";
 import Modal from '../../../Components/Modal/Modal.jsx';
 import axiosRetry from "axios-retry";
 import swal from 'sweetalert2';
+import Joyride from 'react-joyride';
```

Comment:
has this been added to the required dependencies so that when we do npm install everything works?

Reply:
Thank you for bringing that up! I have added all the necessary dependencies for the onboarding tour to the project. This includes any packages needed for the functionality to work properly. Once you run npm install, everything should work seamlessly.

If you encounter any issues with missing dependencies after the installation, please let me know and I’ll address them promptly!